packages/vaex-core/vaex/dataframe.py

```diff
@@ -5894,6 +5894,26 @@ def write(writer):
         else:
             write(to)
 
+    @docsubst
+    def export_feather(self, to, parallel=True, reduce_large=True, compression='lz4', fs_options=None):
+        """Exports the DataFrame to an arrow file using the feather file format version 2
+
+        Feather is exactly represented as the Arrow IPC file format on disk, but also support compression.
+            see also https://arrow.apache.org/docs/python/feather.html
+
+        :param to: filename or file object
+        :param bool parallel: {evaluate_parallel}
+        :param bool reduce_large: If True, convert arrow large_string type to string type
+        :param compression: Can be one of 'zstd', 'lz4' or 'uncompressed'
+        :param fs_options: Placeholder for now
+        :return:
+        """
+        import pyarrow.feather as feather
+        table = self.to_arrow_table(parallel=False, reduce_large=reduce_large)
+        fs_options = fs_options or {}
+        with vaex.file.open(path=to, mode='wb', fs_options=fs_options) as sink:
+            feather.write_feather(table, sink, compression=compression)
```

Comment:
I was pleasantly surprised that this seems to work.  In the [docs](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html) it's only meant to be a string, but seems to support sinks just fine so we can write to s3 etc.

Reply:
Yeah, just tested that, and was able to write to s3.